networkx/algorithms/similarity.py

```diff
@@ -1429,9 +1429,7 @@ def simrank_similarity_numpy(
     newsim = np.eye(adjacency_matrix.shape[0], dtype=np.float64)
     for _ in range(max_iterations):
         prevsim = np.copy(newsim)
-        newsim = importance_factor * np.matmul(
-            np.matmul(adjacency_matrix.T, prevsim), adjacency_matrix
-        )
+        newsim = importance_factor * (adjacency_matrix.T @ prevsim @ adjacency_matrix)
```

Comment:
```suggestion
        newsim = importance_factor * ((adjacency_matrix.T @ prevsim) @ adjacency_matrix)
```
Shouldn't make a difference, but maybe *slightly* more readable since the `matmul` version left no question about order-of-operations.